libs/resources/RespCommandsInfo.json

```diff
@@ -4837,6 +4837,31 @@
       }
     ]
   },
+  {
```

Comment:
Shouldn't these Garnet-specific commands be in playground/CommandInfoUpdater/GarnetCommandsInfo.json?

Reply:
cc @TalZaccai ?

Reply:
Done